Texas A&M University

MediocreWizard opened the pull request:
# Texas A&M University

These files will be available to all teams in the SWCCDC Regional event.

Please ensure that your pull request meets the following requirements:

- [x] This is the only pull request our team has made.
- [x] We have explicit permission to use and distribute these scripts for the game.
- [x] All files meet the requirements outlined in the root README.

### Note
We will only start reviewing pull requests that fully pass all CI jobs.

---
Please provide any additional information or context below:

Commit:
Add scripts & configs

gb1035 commented:
@MediocreWizard don't worry about the "binary-file" check failing. I'll update it to also allow Javascript files. But the line-length check is a legitimate issue.

Commit:
Split long line

Commit:
Split long lines

The software only supports single-line statements in config files [1],
so the lines can't be split while remaining valid.

Split into multiple lines and added a preprocessing step to join the
lines during the build process.

[1] https://www.haproxy.com/documentation/haproxy-configuration-manual/latest/#2.1

MediocreWizard commented:
Looks like I misunderstood what the 400 characters per line meant (I figured it didn't matter since the pr was nowhere near 15000 lines in total). I've fixed the line length issue for the shell script and the config file, though I added a script to rejoin the lines for the config since it only supports single-line statements.

gb1035 commented:
@MediocreWizard Ok, great. Can you merge the latest from our main into this, then all checks should pass then.
